arches/app/models/resource.py

```diff
@@ -500,6 +500,26 @@ def get_documents_to_index(
         document["displayname"] = []
         document["displaydescription"] = []
         document["map_popup"] = []
+        try:
+            document["date_created"] = (
+                models.EditLog.objects.filter(
+                    resourceinstanceid=str(self.resourceinstanceid)
+                )
+                .earliest("timestamp")
+                .timestamp
+            )
+        except:
+            document["date_created"] = None
```

Comment:
The date created is a property on the resource instance itself, so there shouldn't be a need to go to the edit log for this. 

```suggestion
        document["date_created"] = self.createdtime
```